ibm_catalog.json

```diff
@@ -0,0 +1,543 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-vpc",
+      "label": "[Add-ons Beta] Cloud automation for VPC",
+      "product_kind": "solution",
+      "tags": [
+        "network_vpc",
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "support_ibm",
+        "solution"
+      ],
+      "keywords": [
+        "vpc",
+        "terraform",
+        "IaC",
+        "infrastructure as code",
+        "solution"
+      ],
+      "short_description": "Automates VPC deployment on IBM Cloud, offering full configurability and flexibility for diverse workloads.",
+      "long_description": "The VPC deployable architecture deploys a Virtual Private Cloud (VPC) infrastructure without any compute resources, such as Virtual Server Instances (VSI) or Red Hat OpenShift clusters.This is an experimental tile and not suitable for production workloads. Stay here if you want to try an experimental version with the Optional and swappable components capability.",
```

Comment:
can you please add the URLs pointing to the optional and swappable components as well?

Reply:
added